modeling/07_Shoreline_Contamination.Rmd

```diff
@@ -63,6 +68,8 @@ aoi_crs <- st_crs(aoi)
 sc <- st_transform(sc, aoi_crs)
 ```
 
+## View the data and clip the AOI
+
 And plot to see where they are in conjunction with the shapefile
 
 ```{r}
```

Comment:
The `r nrow(sc)` in this chunk does not print the value it prints that verbatim.

Reply:
ugh. stupid line break issue. THANK YOU!